src/models.jl

```diff
@@ -8,6 +8,7 @@ export AbstractModel,
     prognostic_vars,
     auxiliary_vars
 
+import .Domains: coordinates
```

Comment:
im not sure if this needs to be here?

Reply:
This method is extended at the end of the file